### DISCUSSION TITLE
Multiple City Hubs Interacting

### DISCUSSION THREAD

**Discussion by sforment** [pybricks/support#1005] [Q&A]
There are mentions  in the discussions of a method for hub-to-hub communication being available, but the answers are a bit vague and don't point to the solution for this directly. Where can I read about the solution for two hubs interacting with each other? 

To be clear I want to use 2 cityHubs together, one with a color sensor and motor one with only a motor. The idea is to have the motors on both hubs mimics each others behavior depending on what the color sensor connected to only the one hub reads. Is that possible with pybricks?

Thank you!

**Comment by laurensvalk** [comment]
You could try this: https://pybricks.com/projects/tutorials/wireless/hub-to-hub/broadcast/

**Reply by sforment** [reply]
Thanks so much!!!
